src/frontends/ir/tests/frontend_test_basic.cpp

```diff
@@ -0,0 +1,957 @@
+        <layer name="output" type="Result" id="1" version="opset1">
+            <input>
+                <port id="0" precision="FP32">
+                    <dim>1</dim>
+                    <dim>3</dim>
+                    <dim>22</dim>
+                    <dim>22</dim>
+                </port>
+            </input>
+        </layer>
+    </layers>
+    <edges>
+        <edge from-layer="0" from-port="0" to-layer="1" to-port="0"/>
+    </edges>
+</net>
+)V0G0N";
+
+    std::shared_ptr<ov::Model> modelv10;
+    ov::RTMap rtInfoV10;
+    uint64_t version;
+
+    ASSERT_NO_THROW(modelv10 = getWithIRFrontend(testNetworkV10));
+    ASSERT_TRUE(!!modelv10);
+    ASSERT_NO_THROW(rtInfoV10 = modelv10->get_rt_info());
+    ASSERT_NO_THROW(version = rtInfoV10["version"].as<int64_t>());
+    ASSERT_EQ(10, version);
+
+    std::shared_ptr<ov::Model> modelRef;
+    {
+        auto parameter = std::make_shared<ov::opset1::Parameter>(ov::element::f32, ov::Shape{1, 3, 22, 22});
+        parameter->set_friendly_name("input");
+        auto result = std::make_shared<ov::opset1::Result>(parameter);
+        result->set_friendly_name("output");
+        modelRef = std::make_shared<ov::Model>(ov::NodeVector{result}, ov::ParameterVector{parameter});
+    }
+
+    const auto fc = FunctionsComparator::with_default()
+                        .enable(FunctionsComparator::ATTRIBUTES)
+                        .enable(FunctionsComparator::PRECISIONS)
+                        .enable(FunctionsComparator::RUNTIME_KEYS)
+                        .enable(FunctionsComparator::NAMES)
+                        .enable(FunctionsComparator::CONST_VALUES);
+    const auto res = fc.compare(modelv10, modelRef);
+    EXPECT_TRUE(res.valid) << res.message;
+}
+
+TEST_F(IRFrontendTests, elementary_network_reading_v9) {
+    std::string testNetworkV9 = R"V0G0N(
+<net name="Network" version="9">
+    <layers>
+        <layer name="input" type="Parameter" id="0" version="opset1">
+            <data element_type="f32" shape="1,3,22,22"/>
+            <output>
+                <port id="0" precision="FP32">
+                    <dim>1</dim>
+                    <dim>3</dim>
+                    <dim>22</dim>
+                    <dim>22</dim>
+                </port>
+            </output>
+        </layer>
+        <layer name="output" type="Result" id="1" version="opset1">
+            <input>
+                <port id="0" precision="FP32">
+                    <dim>1</dim>
+                    <dim>3</dim>
+                    <dim>22</dim>
+                    <dim>22</dim>
+                </port>
+            </input>
+        </layer>
+    </layers>
+    <edges>
+        <edge from-layer="0" from-port="0" to-layer="1" to-port="0"/>
+    </edges>
+</net>
+)V0G0N";
+
+    std::shared_ptr<ov::Model> modelv9;
+    ASSERT_THROW(modelv9 = core.read_model(testNetworkV9, ov::Tensor()), ov::Exception);
+    ASSERT_FALSE(!!modelv9);
+}
+
+TEST_F(IRFrontendTests, networ_with_missing_weights) {
```

Comment:
Typo + can you replace `network` to `model`?
```suggestion
TEST_F(IRFrontendTests, network_with_missing_weights) {
```